server/metrics.go

```diff
@@ -0,0 +1,42 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/openfresh/plasma/config"
+	"github.com/openfresh/plasma/metrics"
+	"go.uber.org/zap"
+)
+
+type metricsHandler struct {
+	accessLogger *zap.Logger
+	errorLogger  *zap.Logger
+	config       config.Config
+	mux          *http.ServeMux
+}
+
+func (h metricsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	h.mux.ServeHTTP(w, r)
+}
+
+func NewMetricsHandler(opt Option) metricsHandler {
```

Comment:
__[golint]__
* [golint] exported function NewMetricsHandler should have comment or be unexported
* [golint] exported func NewMetricsHandler returns unexported type server.metricsHandler, which can be annoying to use

<!-- Comment by [SideCI](https://sideci.com) -->